seatunnel-core/seatunnel-flink-starter/src/main/java/org/apache/seatunnel/core/starter/flink/execution/FlinkExecution.java

```diff
@@ -90,6 +111,33 @@ private void registerPlugin() {
 
         pluginsJarDependencies.forEach(url -> FlinkCommon.ADD_URL_TO_CLASSLOADER.accept(Thread.currentThread().getContextClassLoader(), url));
 
-        flinkEnvironment.registerPlugin(pluginsJarDependencies);
+        jarPaths.addAll(pluginsJarDependencies);
+    }
+
+    private Config registerPlugin(Config config, List<URL> jars) {
+        config = this.parseConfig(config, ConfigUtil.joinPath("env", "pipeline", "jars"), jars);
+        return this.parseConfig(config, ConfigUtil.joinPath("env", "pipeline", "classpaths"), jars);
```

Comment:
Change the doc to tell user the new way add jar. Also this function should support spark later.

Reply:
> Change the doc to tell user the new way add jar. Also this function should support spark later.

This change does not require a change in usage,it's just the way the jar is injected that has changed.

Reply:
But you support add jar use `env.pipeline.jars` in config

Reply:
> Change the doc to tell user the new way add jar. Also this function should support spark later.
Spark uses the command line,i don't think it needs changing.

Reply:
> But you support add jar use `env.pipeline.jars` in config
This is only used by flink,does spark need this way?

Reply:
> But you support add jar use `env.pipeline.jars` in config

I see.